.github/workflows/ci.yaml

```
@@ -5,7 +5,17 @@ on:
   - pull_request
 
 jobs:
-  ci:
-    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@dec0880059017dac7facf100435c5737bf1386c8
+  pathogen-ci:
+    strategy:
+      matrix:
+        runtime: [docker, conda]
+    permissions:
+      id-token: write
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@master
+    secrets: inherit
     with:
-      workflow-root: phylogenetic
\ No newline at end of file
+      runtime: ${{ matrix.runtime }}
+      run: |
+        nextstrain build \
+          phylogenetic \
+          --configfile profiles/ci/profiles_config.yaml
```

Ah, looks like we missed a [couple lines](https://github.com/nextstrain/mpox/blob/0ff5ce863c1c8c51e0daf164bfbba54d5b7fef24/.github/workflows/ci.yaml#L25-L31) when we were copying over the CI workflow from mpox

```suggestion
          --configfile profiles/ci/profiles_config.yaml
      artifact-name: output-${{ matrix.runtime }}
      artifact-paths: |
        phylogenetic/auspice/
        phylogenetic/results/
        phylogenetic/benchmarks/
        phylogenetic/logs/
        phylogenetic/.snakemake/log/
```

The `artifact-name` ensures that we produce separate artifacts for the docker and conda builds
The `artifact-paths` is used to specify paths within the phylogenetic directory.